src/+replab/CharacterTable.m

```diff
@@ -0,0 +1,112 @@
+classdef CharacterTable < replab.Obj
+% Describes the character table of a group
+
+    properties (SetAccess = protected)
+        group % replab.Group: group represented by character table
+        irreps % cell of replab.Rep: irreducible representation
+        classes % cell of replab.ConjugacyClass: conjugacy classes
+        chars % matrix(nirreps,nclasses): matrix of characters
+        table % replab.str.Table: table object for display purposes
+    end
+    
+    methods
+
+        function self = CharacterTable(group)
+            self.group = group;
```

Comment:
Indeed, we can precompute stuff; but that's the responsibility of the `group.characterTable` method.

What we will do: for instances of `SymmetricGroup`, we know what to do directly as we follow the standard presentation. For other groups, if they can be recognized quickly, we'll find the standard generators and then call Andrew's code.

Can you file an issue so that we don't forget to do this?